recipes/dexplot/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "dexplot" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1794c4d08ed3cd16d257937c224b4c7757f69bd4dd4d7c914f9bd9ec275da5da
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This could be `noarch: python`

Reply:
Thanks for the replies. I used `conda skeleton pypi dexplot` to make the meta.yaml. I suppose it's best not to use it?

And for `noarch: python`, it states in https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python that there should be "python version specific requirements".

Reply:
I think the preferred tool these days is https://github.com/marcelotrevisani/grayskull. Grayskull does a pretty good job in my experience and usually only requires minor tweeks to get the recipe in shape.

The "python version specific requirements" just means that you don't have any requirements that are specific to the version of python that you're building against or for the platform (e.g win64 vs linux64). I think in your case, you should be fine using `noarch: python`.